internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -761,7 +886,8 @@ func NewGrafanaService(cr *operatorv1beta1.Cryostat) *corev1.Service {
 		Spec: corev1.ServiceSpec{
 			Type: *config.ServiceType,
 			Selector: map[string]string{
-				"app": cr.Name,
+				"app":       cr.Name,
+				"component": "grafana",
```

Comment:
I noticed the selector with component doesn't work. All 3 services select the same pod, but the deployment's pod template has `component: cryostat`. We end up with the Grafana container running normally, but the service doesn't route traffic to it.

Reply:
Ah, right...

Reply:
What do you think is the best way through this then? Should I set the `component` for the Grafana service to just be `cryostat`?

Reply:
We could do that, or just skip the `component` label and use `app: cr.Name + "-reports"` for the reports deployment and service.